frontend/src/features/user/transfer-ownership/TransferOwnershipModal.tsx

```diff
@@ -38,7 +43,22 @@ type TransferOwnershipInputs = {
   email: string
 }
 
-const useModalState = ({ onClose, reset, trigger }) => {
+const useModalState = ({
+  onClose,
+  reset,
+  trigger,
+}: {
+  onClose: () => void
+  reset: UseFormReset<TransferOwnershipInputs>
+  trigger: UseFormTrigger<TransferOwnershipInputs>
+}): {
+  page: number
```

Comment:
This is a lil hard to read should we refactor it into `useModalStateReturn` 

then 
```
}) : useModalStateReturn => {
```

Reply:
Done :) [5d4e67b](https://github.com/opengovsg/FormSG/pull/6802/commits/5d4e67b193fd04f18910d7e00315a5bdab096531)